File: app/src/main/java/com/hustunique/morii/home/PresenterMain.java
package com.hustunique.morii.home;

import static com.hustunique.morii.util.MyApplication.musicDiaryList;

public class PresenterMain implements HomePageContract.IPresenter {
    HomePageContract.IView view;

    PresenterMain(HomePageContract.IView iViewMain){
        this.view = iViewMain;
    }

    @Override
    public void getUI(){
        view.GetRecyclerView(musicDiaryList);
    }


}

File: app/src/main/java/com/hustunique/morii/home/PresenterMain.kt
package com.hustunique.morii.home

import com.hustunique.morii.util.MyApplication

class PresenterMain internal constructor(var view: HomePageContract.IView) :
    HomePageContract.IPresenter {
    override val uI: Unit
        get() {
            view.setRecyclerView(MyApplication.Companion.musicDiaryList)
        }
}